frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -617,6 +618,16 @@ export const surveyLogic = kea<surveyLogicType>([
                 return !!(survey.start_date && !survey.end_date)
             },
         ],
+        descriptionContentType: [
+            (s) => [s.survey],
+            (survey) => (questionIndex: number) => {
+                return survey.questions[questionIndex].descriptionContentType
+            },
+        ],
+        thankYouMessageDescriptionContentType: [
+            (s) => [s.survey],
+            (survey) => survey.appearance.thankYouMessageDescriptionContentType,
+        ],
```

Comment:
Given that  this is just a simple property access, I wouldn't use a selector here, but access it directly in `SurveyEdit`: `const { thankYouMessageDescriptionContentType } = survey.appearance;`